connect-streams-pipeline/docs/index.rst

```diff
@@ -129,7 +129,7 @@ Run the demo
 Example 1: Kafka console producer -> Key:String and Value:String
 ----------------------------------------------------------------
 
-- Command line ``confluent local produce`` produces ``String`` keys and ``String`` values to a Kafka topic.
+- Command line ``confluent local services kafka produce`` produces ``String`` keys and ``String`` values to a Kafka topic.
```

Comment:
Good catch @brianstrauch .  In this file there are two more references to `confluent local produce`, could you please update those as well?

Reply:
Sure! I didn't change them originally because they looked like version notes, and I didn't know if they should be left unedited for historical purposes.